definitions/features/puppet_server.rb

```diff
@@ -2,15 +2,42 @@ class Features::PuppetServer < ForemanMaintain::Feature
   metadata do
     label :puppet_server
 
-    # We only check puppetserver and not puppet-server, as puppet-server
-    # is a part of httpd and relies on httpd service to restart, therefore
-    # not requiring a separate service to restart
     confine do
-      find_package('puppetserver')
+      find_package('puppet-server') || find_package('puppetserver')
```

Comment:
@mbacovsky, @iNecas,

I can see there is one more feature called [`puppet` ](https://github.com/theforeman/foreman_maintain/blob/master/definitions/features/puppet.rb) and in which there is similar confine block i.e

```
confine do
   find_package('puppetserver')
end
```
In that feature we have method `config_files`. Should I move this method to `puppet_server` feature?
and instead of checking `find_package('puppetserver')` may I add `find_package('puppet')` for `puppet` feature?

please confirm.

Reply:
I'm okw with merging, but it should be called `:puppet_server` - as might want to add the `puppet_agent` in the future.

Reply:
Thank you @mbacovsky and @iNecas. I will update PR with above changes.

Reply:
+1 to what @iNecas said